tests/stack_functions.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+
+import numpy as np
+
+import cunumeric as num
+
+
+def run_test(arr, test_routine):
+    for routine in test_routine:
+        print_msg = f"np.{routine}((array({arr[0].shape})*3)"
+        b = getattr(np, routine)(arr)
```

Comment:
Looks like you copy-pasted the test for `concatenate` from this one... is there a reason that you didn't test `concatenate` together in this test? you may want to consider merging the tests if possible.